src/systems/systems/src/simulation/update_context.rs

```diff
@@ -12,6 +15,10 @@ pub struct UpdateContext {
     pub indicated_altitude: Length,
     pub ambient_temperature: ThermodynamicTemperature,
     pub is_on_ground: bool,
+    pub acc_long: Acceleration,
```

Comment:
**question**: What type of acceleration is `acc_long` representing?
**suggestion**: Improve the field name.